.npmignore

```diff
@@ -0,0 +1 @@
+coverage
```

Comment:
Why don't you prevent all system files to be pushed to the npm registry?

```
.vscode
.DS_Store
node_modules
package-lock.json
yarn.lock

coverage
```

Reply:
package-lock.json and node_modules are automatically ignored, I will add the rest in